samples/v2/pipeline_container_no_input.yaml

```diff
@@ -0,0 +1,27 @@
+# PIPELINE DEFINITION
+# Name: v2-container-component-no-input
```

Comment:
Could we add a README briefly describing each of these pipeline examples? Probably out of scope for this PR/task and should be a follow up task

Reply:
Actually this PR itself might get closed if its deemed unnecessary. I like that idea though of having a README briefly describing each of the pipeline examples.